### DISCUSSION TITLE
Voltage reading is false

### DISCUSSION THREAD

**Discussion by GiannisDorokidis** [arendst/Tasmota#22233] [Q&A] [answered]
Im setting up a gosound sp111 and the voltage reading seems to be off by 100 volts.The normal one should be at 220-240 volts
![image](https://github.com/user-attachments/assets/83a0e0e0-aa05-4537-9dbe-633594118d5b)

Here is the template i am using 
`{"NAME":"Gosund SP111 v1.1","GPIO":[320,0,576,0,2656,2720,0,0,2624,32,0,224,0,0],"FLAG":0,"BASE":45}`

**Comment by sfromis** [comment, marked as answer]
You most likely need https://tasmota.github.io/docs/Power-Monitoring-Calibration/
